src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java

```diff
@@ -35,15 +40,20 @@
  * Release test that verifies basic reading from a YugabyteDB database and
  * writing to Kafka and then further to a PostgreSQL sink database
  *
- * @author Isha Amoncar, Vaibhav Kushwaha (vkushwaha@yugabyte.com)
+ * @author Isha Amoncar, Sumukh Phalgaonkar, Vaibhav Kushwaha (vkushwaha@yugabyte.com)
  */
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
Why is this required? I thought JUnit was able to understand the simple `@Order` annotation without specifying any orderer class.

Reply:
No `@Order` was not working without this. Earlier inserts where happening in `@BeforeAll` block so the order of running tests didn't matter.  I moved the inserts inside first test hence order of execution became important.